pkgs/top-level/all-packages.nix

```diff
@@ -16720,6 +16720,12 @@ with pkgs;
 
   samurai = callPackage ../development/tools/build-managers/samurai { };
 
+  muon = callPackage ../development/tools/build-managers/muon { };
+  muonStandalone = muon.override {
+    embedSamurai = true;
+    buildDocs = false;
```

Comment:
Why do we not need docs here?

Reply:
The idea is to use it as a minimal standalone binary that requires only a C compiler (and some libs written in C).

The docs require Python with pyYAML libs.